cmd/utils/flaggroup.go

```diff
@@ -141,6 +141,8 @@ var FlagGroups = []FlagGroup{
 			ChainDataFetcherKafkaTopicResourceFlag,
 			ChainDataFetcherKafkaReplicasFlag,
 			ChainDataFetcherKafkaPartitionsFlag,
+			ChainDataFetcherKafkaMaxMessageBytes,
```

Comment:
Could you change `ChainDataFetcherKafkaMaxMessageBytes` to `ChainDataFetcherKafkaMaxMessageBytesFlag`?

Reply:
Reflected. 9f4dd7b64c2024ad88c91a92a7e0a3665d7c231a